modules/twinkleblock.js

```diff
@@ -251,10 +248,8 @@ Twinkle.block.callback.change_block64 = function twinkleblockCallbackChangeBlock
 	var priorName = relevantUserName;
 	if ($block64.is(':checked')) {
 		relevantUserName = Morebits.ip.get64(mw.config.get('wgRelevantUserName'));
-		$form.find('[name=actiontype][value=template]').prop('disabled', true).prop('checked', false);
 	} else {
 		relevantUserName = mw.config.get('wgRelevantUserName');
-		$form.find('[name=actiontype][value=template]').prop('disabled', Morebits.ip.isRange(relevantUserName)).prop('checked', !Morebits.ip.isRange(relevantUserName));
```

Comment:
Again, not critical now, but we will need something once `wgRelevantUserName` is defined for IP ranges.  At that point, if blocking a range smaller than a /64 (/65, etc.), the `block64` will be offered, but we don't want to offer a template in that case.

Like below with the template, though, it'd awkwardly depend on the original IP, so maybe something like:

```js
// No templates for ranges, but if the original user is a single IP, offer the option
// (done separately in Twinkle.block.callback.issue_template)
var originalIsRange = Morebits.ip.isRange(mw.config.get('wgRelevantUserName'));
$form.find('[name=actiontype][value=template]').prop('disabled', originalIsRange).prop('checked', !originalIsRange);
```